### PR TITLE
[SRVKE-1155] Add the KEDA autoscaling for KafkaSource docs

### PR DESCRIPTION
**Pull request by rh-max**
Version(s):
`serverless-docs-1.33`+

Issue:
https://issues.redhat.com/browse/SRVKE-1155

Link to docs preview:
https://77485--ocpdocs-pr.netlify.app/openshift-serverless/latest/eventing/event-sources/serverless-kafka-developer-source.html#serverless-kafka-source-autoscale-keda_serverless-kafka-developer-source

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/serverless-kafka-source-autoscale-keda.adoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * eventing/event-sources/serverless-kafka-developer-source.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-kafka-source-autoscale-keda_{context}"]
+= Configuring KEDA autoscaling for KafkaSource
+
+You can configure Knative Eventing sources for Apache Kafka (KafkaSource) to be autoscaled using the Custom Metrics Autoscaler (KEDA) operator.
```
Comment (skrthomas) [inline modules/serverless-kafka-source-autoscale-keda.adoc:9]: Just want to check, does this operator need to be capitalized, like Operator? I know a lot of operators usually are capitalized O in the doc.

**Review by skrthomas — COMMENTED** [review]
One small question. Nice work!

File: modules/serverless-kafka-source-autoscale-keda.adoc (hunk 1 above)
Comment (rh-max) [reply modules/serverless-kafka-source-autoscale-keda.adoc]: Thanks! It does, and I expanded the description to be more precise: `the Custom Metrics Autoscaler Operator, which is based on the Kubernetes Event Driven Autoscaler (KEDA)`.

File: modules/serverless-kafka-source-autoscale-keda.adoc [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+* The {ServerlessOperatorName}, Knative Eventing, and the `KnativeKafka` custom resource are installed on your cluster.
+
+.Procedure
+
+. In the `KnativeKafka` custom resource, enable KEDA scaling:
++
+.Example YAML
+[source,yaml]
+----
```
Comment (mgencur) [inline modules/serverless-kafka-source-autoscale-keda.adoc:24]: There's "config" missing in the spec, the example should be like this:
```
apiVersion: operator.serverless.openshift.io/v1alpha1
kind: KnativeKafka
metadata:
  name: knative-kafka
  namespace: knative-eventing
spec:
  config:
    kafka-features:
      controller-autoscaler-keda: enabled
```

Comment (rh-max) [reply modules/serverless-kafka-source-autoscale-keda.adoc:24]: Thanks, fixed!

**Comment by mgencur** [comment]
LGTM from QE

**Pull request merged**